packages/test/package.json

```diff
@@ -80,7 +80,7 @@
     "@types/react": "^16.8.4 || ^17.0.0",
     "react": "^16.8.4 || ^17.0.0",
     "redux": "^4.0.0",
-    "rest-hooks": "^5.0.0-beta.5"
+    "rest-hooks": "^5.0.11"
```

Comment:
Good question. Basically there *is* an additional export in  #643 - but it's not a new feature but a fix for something that should already work. Not having that export is broken. Furthermore, there is no sense in making a 5.1 docs site cut as there's no correct way to do redux subtrees in the other version and everything else will be the same.

So the question is how important is it to minor bump on *any* export added?

Reply:
Are you saying zero part of the package works without that export?

Reply:
No, only this one use case is affected - redux integration

Reply:
then for some users, it's a bug fix, but for some users, it's not a bugfix at all, just a new feature - which i'd say makes it semver-minor.

obviously it's not a catastrophe either way :-)

Reply:
okay, gonna land it in this form, since this makes docs more readable. the version support matching should still protect people whether it's patch or minor. users that don't use redux shouldn't care either way